src/main/proto/ga4gh/read_service.proto

```diff
@@ -91,6 +103,52 @@ message GetReadGroupSetRequest {
   string read_group_set_id = 1;
 }
 
+// ******************  /readgroups  *********************/
+
+// This request maps to the body of `POST /readgroups/search` as JSON.
+
+
+message SearchReadGroupsRequest {
+  // The dataset to search.
```

Comment:
As long as proto2 messages don't get imported or vice versa in mixed form - this is from [the same page](https://developers.google.com/protocol-buffers/docs/proto):

> Using proto3 Message Types
>
> It's possible to import [proto3](https://developers.google.com/protocol-buffers/docs/proto3) message types and use them in your proto2 messages, and vice versa. However, proto2 enums cannot be used in proto3 syntax. 

I'll add a note the other post in this PR to make it more clear in case people search for it, and try to use it.